qc_grader/grader/api.py

```diff
@@ -132,15 +134,18 @@ def get_submission_endpoint(
 
 def get_access_token() -> Optional[str]:
     iqx_token = os.getenv('QXToken')
+    saved_runtime_token = QiskitRuntimeService.saved_accounts()['default-ibm-quantum']['token']
     if iqx_token is None:
-        return None
+        if saved_runtime_token is not None:
+            iqx_token = saved_runtime_token
+        else:
+            return None
```

Comment:
if there is no saved account or for some reason no `default-ibm-account` then this would result in error. also if we have a `QXToken` we can skip checking saved accounts: 

```suggestion
    iqx_token = os.getenv('QXToken')
    if iqx_token is None:
        iqx_token = QiskitRuntimeService.saved_accounts().get('default-ibm-quantum', {}).get('token')
   return iqx_token
```